presto-docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,92 @@
+on a self-managing database layer, while being optimized for lake engines and regular batch
+processing. By adopting Multiversion Concurrency Control (MVCC), where compaction action
+merges logs and base files to produce new file slices and cleaning action gets rid of
+unused/older file slices to reclaim space on the file system, Hudi can support two types of
+tables: Copy-on-Write and Merge-on-Read.
+
+Hudi table state is maintained in commit metadata files, which is essentially a timeline of
+all actions performed on the table at different instants of time that helps provide
+instantaneous views of the table, while also efficiently supporting retrieval of data in
+the order of arrival. To learn more about Hudi, please check out the `overview <https://hudi.apache.org/docs/overview/>`_
+and `quickstart <https://hudi.apache.org/docs/quick-start-guide/>`_
+
+This connector enables querying Hudi tables synced to Hive metastore. The connector uses
+the metastore only to track partition locations. It makes use of the underlying Hudi file
+system and input formats to list data files. Hudi connector runs a partition scanner and
+split generator asynchronously. As the name suggests, the partition scanner will scan all
+the partitions and split generator will list files and generate splits for each pruned
+partition. To learn more about the design of the connector, please check out `RFC-40 <https://github.com/apache/hudi/blob/master/rfc/rfc-44/rfc-44.md>`_.
+
+--------
+Requirements
+--------
+To use Hudi, we need:
+
+* Network access from the Presto coordinator and workers to the distributed object storage.
+* Access to a Hive metastore service (HMS).
+* Network access from the Presto coordinator to the HMS. Hive metastore access with the Thrift protocol defaults to using port 9083.
+
+--------
+Configuration
+--------
+Hudi supports the same metastore configuration properties as the Hive connector. At a minimum, following connector properties must be set in the
+hudi.properties
+file inside
+<presto_install_dir>/etc/catalog
+directory.
+
+======================================= ============================================= ===========
+Property Name                           Description                                   Default
+======================================= ============================================= ===========
+``hudi.metadata-table-enabled``         Fetch the list of file names and sizes from   false
+                                        Hudi's metadata table rather than storage.
+======================================= ============================================= ===========
+
+-----------
+SQL support
+-----------
+Currently, the connector only provides read access to data in the Hudi table that has been synced to
+Hive metastore. Once the catalog has been configured as mentioned above, users can query the tables
+as usual like Hive tables::
+
+    presto> use hudi.default;
```

Comment:
Let's make the examples more clear. You may format them as [Iceberg's Example Queries](https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/iceberg.rst#example-queries) section does

Reply:
fix it